### DISCUSSION TITLE
KM_estimate with censored data

### DISCUSSION THREAD

**Discussion by martinapili** [CamDavidsonPilon/lifelines#1523] [Q&A]
Hello, I am using KaplanMeierFitter() to fit a right-censored dataset and I am struggling to interpret the KM estimate from the resulting survival function: 

![image](https://github.com/CamDavidsonPilon/lifelines/assets/41012160/5673b6c1-3de1-4b8f-9519-b1941f9a245a)

Looking at year 10 I see 20% survival probability which doesn't match the at_risk counts in the table below. I would have expected a survival probability of 22/(844 + 22) = 2.5%. Am I misinterpreting the way the KM_estimate is calculated? Note that if I exclude the censored data points the (At_risk)/(Events + At risk) formula works as expected.

Thanks a lot!

**Comment by CamDavidsonPilon** [comment]
Hi @martinapili, 

The table below isn't the full picture. You need the deltas of columns to compute the survival curve (km estimate). Furthermore, the KM estimate is an accumulation of probabilities (probs not dying in period 0 * probs not dying in period 1 * ... ). 

One thing to always remember is that we are estimating S(t) = P(surviving past time t). So 20% survival at year 10 suggest that a subject has a 20% chance to survive past 10 years. It's not unbelievable since your data has lots of censored subjects, and fewer and fewer at-risk subjects die in years 6 to 10 (expect past 10 years - there's a cliff of death).
